src/styles/index.scss

```diff
@@ -1,3 +1,10 @@
-body {
-  margin: 0;
+* {
+  box-sizing: border-box;
 }
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the universal selector (*) is not recommended for setting 'box-sizing' because it can be heavy on performance. Instead, it's better to set 'box-sizing' directly on 'html' and inherit it for all other elements.